### PR TITLE
Add thriftrw.install

### PR DESCRIPTION
**Pull request by abhinav**
Use case explained in docstring, but this is related to that conversation we had regarding `from <dynamically generated module> import foo`. For TCollector and VCR in the TChannel library, we'll want them to install the Thrift files as submodules in the same way they currently have the Thrift generated code under a subdirectory.

Soliciting opinions on whether you think this is too much magic.

@breerly @blampe

### CODE REVIEW

**Comment by abhinav** [comment]
Keep in mind that this is basically,

    thrift_file = os.path.join(os.path.dirname(__file__), 'my_service.thrift')
    sys.modules[__name__ + '.my_service'] = thriftrw.load(thrift_file)

**Comment by breerly** [comment]
I'm ok with it. Let's get @blampe's opinion

File: docs/overview.rst [hunk 1]
```diff
@@ -16,6 +16,39 @@ the ``.thrift`` file.
...
+maps to that generated module. This can be remedied by using
+:py:func:`thriftrw.install`.
+
+.. code-block:: python
+
+    # some_module.py
+
+    import thriftrw
+
+    thriftrw.install('keyvalue.thrift')
```
Comment (blampe) [inline docs/overview.rst]: Would it be possible to write `keyvalue = thriftrw.install(...)` to make the magic more obvious?

Comment (abhinav) [inline docs/overview.rst]: True. Updated.

**Pull request merged**